json/.gitignore

```diff
@@ -0,0 +1,42 @@
+.gradle
```

Comment:
.gitignore file should not exist in submodule folders
Please move it to the root folder with relevant path changes if needed.